.travis.yml

```diff
@@ -2,6 +2,7 @@ language: python
 python:
   - "2.7"
   - "3.6"
+  - "3.7"
```

Comment:
This won't work without sudo and xenial https://github.com/travis-ci/travis-ci/issues/9069